Gemfile

```diff
@@ -174,7 +174,7 @@ group :development do
   gem 'rubocop-rspec_rails', require: false
 
   # Annotates modules with schema
-  gem 'annotaterb', '~> 4.13'
+  gem 'annotaterb', '~> 4.13', require: false
```

Comment:
That would change the workflow when adding new models/new attributes to models, right?

Reply:
The genrated `.rake` file from the gem does its own `require` when in dev env, so I think the flow is the same (or, at minimum, when I run the command, it still runs/works)